Objects/genericaliasobject.c

```diff
@@ -383,22 +384,45 @@ ga_hash(PyObject *self)
     return h0 ^ h1;
 }
 
+#define SET_ORIG_CLASS(obj, alias)  \
```

Comment:
Can't this be a function? Then the three functions can end with `return set_orig_class(obj, self)`.